.gitignore

```diff
@@ -15,4 +15,6 @@
 /typings
 /test-results
 /playwright-report
-/graphql-voyager-*.tgz
\ No newline at end of file
+/graphql-voyager-*.tgz
+/.idea
+.DS_Store
```

Comment:
Please add those to global ignore
https://github.com/graphql-kit/graphql-voyager/blob/fbc5abb585193a429668c106958dfd201c3e0287/.gitignore#L1-L6